tools/Microsoft.AspNetCore.Server.Kestrel.LibuvCopier/Program.cs

```diff
@@ -11,30 +11,8 @@ public void Main(string[] args)
         {
             try
             {
-                var packagesFolder = Environment.GetEnvironmentVariable("DNX_PACKAGES");
-
-                if (string.IsNullOrEmpty(packagesFolder))
-                {
-                    var dnxFolder = Environment.GetEnvironmentVariable("DNX_HOME") ??
-                                    Environment.GetEnvironmentVariable("DNX_USER_HOME") ??
-                                    Environment.GetEnvironmentVariable("DNX_GLOBAL_HOME");
-
-                    var firstCandidate = dnxFolder?.Split(';')
-                                                  ?.Select(path => Environment.ExpandEnvironmentVariables(path))
-                                                  ?.Where(path => Directory.Exists(path))
-                                                  ?.FirstOrDefault();
-
-                    if (string.IsNullOrEmpty(firstCandidate))
-                    {
-                        dnxFolder = Path.Combine(GetHome(), ".dnx");
-                    }
-                    else
-                    {
-                        dnxFolder = firstCandidate;
-                    }
-
-                    packagesFolder = Path.Combine(dnxFolder, "packages");
-                }
+                var packagesFolder = Environment.GetEnvironmentVariable("DNX_PACKAGES") ??
```

Comment:
@davidfowl Should I even be looking at `$DNX_PACKAGES` anymore? Should I just do like you did in https://github.com/aspnet/dnx/commit/a440f9ae3461d6a9f078cbcf6b741d01e7cec95c and get rid of the GetHome method?

             var profileDirectory = Environment.GetEnvironmentVariable("USERPROFILE");
             if (string.IsNullOrEmpty(profileDirectory))
             {
                  profileDirectory = Environment.GetEnvironmentVariable("HOME");
             }
  		  
             return Path.Combine(profileDirectory, ".nuget", "packages");